src/common/Tile/styled.js

```diff
@@ -0,0 +1,188 @@
+import styled, { css } from "styled-components";
+
+export const StyledTile = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 324px;
+  min-height: 650px;
+  border-radius: 5px;
+  background-color: white;
+  padding: 16px;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+  justify-self: center;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
+    flex-direction: row;
+    width: 288px;
+    min-height: 201px;
+  }
+
+  ${({ horizontal }) =>
+    horizontal &&
+    css`
+      flex-direction: row;
+      width: 1368px;
```

Comment:
Może lepiej max-width? Bo na ten moment przy zmniejszeniu ekranu pojawia się suwak na dole i dopiero po przekroczeniu "mobileMax" jest mniejszy kafelek, a myślę że w stanie pośrednim cały Tile powinien się zwężać tak, żeby wszystko mieściło się na ekranie.

Reply:
Racja, tutaj Tomek też zwrócił uwagę, tutaj być może i w ogóle i width i max-width mogłoby wypaść, bo będzie to warunkował pewnie komponent nadrzędny, chociaż w sumie max-width mógłby zostać ;)